Makefile

```diff
@@ -309,3 +309,12 @@ proto-swagger-gen:
 	@sh ./proto/scripts/protoc-swagger-gen.sh
 
 .PHONY: proto-gen proto-doc proto-swagger-gen
+
```

Comment:
Running this target, produced a different `contrib/shell_completion/_gaiad` file. Should update that file for this PR.

Reply:
Hey @mmulji-ic have tested this on my local machine and it is indeed updating the _gaiad file rather than creating a new one.

Is there any specific step you are following? so that  i can reproduce this on my machine?

Reply:
Hi @tokamak-git the file generated on my system was different, was wondering if this was system related or just an older generation of the completions file.